tasks/78_valid-braces/valid-braces.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @description
+ * validates if parenthesis is balanced
+ * @example
+ * // returns true
+ * validBraces('[]{}()')
+ * @param {String} str
+ * @return {boolean}
+ */
+export function validBraces(str) {
+  const stackForBraces = [];
```

Comment:
redundant code

Reply:
what do you mean redundant? you mean that the stack redundant, or the whole logic is off?